x-pack/platform/plugins/shared/stack_connectors/public/connector_types/thehive/constants.ts

```diff
@@ -112,3 +112,117 @@ export const tlpOptions = [
     }),
   },
 ];
+
+export const templateOptions = [
+  {
+    value: 0,
+    text: i18n.translate(
+      'xpack.stackConnectors.components.thehive.eventSelectTemplate1OptionLabel',
+      {
+        defaultMessage: 'Build Your Own',
+      }
+    ),
+  },
+  {
+    value: 1,
+    text: i18n.translate(
+      'xpack.stackConnectors.components.thehive.eventSelectTemplate2OptionLabel',
+      {
+        defaultMessage: 'Compromised User Account Investigation',
+      }
+    ),
+  },
+  {
+    value: 2,
+    text: i18n.translate(
+      'xpack.stackConnectors.components.thehive.eventSelectTemplate3OptionLabel',
+      {
+        defaultMessage: 'Malicious File Analysis',
+      }
+    ),
+  },
+  {
+    value: 3,
+    text: i18n.translate(
+      'xpack.stackConnectors.components.thehive.eventSelectTemplate4OptionLabel',
+      {
+        defaultMessage: 'Suspicious Network Activity',
+      }
+    ),
+  },
+];
+
+export const bodyOptions = [
```

Comment:
If it is only used for tests better to be moved to the test files or to a `mock.ts` file. Same for `testBodyOptions`.

Reply:
Ok, I saw they are being used for the component. Better to use a record with key-value pairs. It is very hard to understand what each value means and how it is being used.